pandas/parser.pyx

```diff
@@ -13,7 +13,10 @@ from cpython cimport (PyObject, PyBytes_FromString,
                       PyUnicode_Check, PyUnicode_AsUTF8String,
                       PyErr_Occurred, PyErr_Fetch)
 from cpython.ref cimport PyObject, Py_XDECREF
-from io.common import CParserError, DtypeWarning, EmptyDataError
+# XXX: It's annoying that we have to import both CParserError
+# and ParserError. Unfortunately, backwards compatibility is
+# the higher calling right now.
```

Comment:
Can you keep this a bit more 'dry' and simply say "import CParserError as alias of ParserError for backward compatibility"

you can maybe also put the one import of CParserError on a separate line to make this more clear